flexmeasures/data/models/planning/storage.py

```diff
@@ -455,7 +455,16 @@ def compute(self, skip_validation: bool = False) -> pd.Series | None:
         if self.round_to_decimals:
             storage_schedule = storage_schedule.round(self.round_to_decimals)
 
-        return storage_schedule
+        if self.return_multiple:
+            return [
+                {
+                    "name": "storage_schedule",
+                    "sensor": sensor,
+                    "data": -storage_schedule,
```

Comment:
Our schedulers should be using the convention: `consumption = +`. Switching here is not consistent behaviour.

Reply:
You're right! I've removed the sign.